### PR TITLE
Update to Blog

### PR DESCRIPTION
**Pull request by JRPearson500**

### CODE REVIEW

**Comment by JRPearson500** [comment]
Added further details on quality metrics off the back of Dom's and Jennifer's work and references to more recent work.

File: synthetic.md [hunk 1]
```diff
@@ -100,16 +102,18 @@ At this point it is useful to again consider if synthetic data is the right solu
...
-* [Voas Williamson statistic](https://onlinelibrary.wiley.com/doi/abs/10.1002/1099-1220%28200009/10%296%3A5%3C349%3A%3AAID-IJPG196%3E3.0.CO%3B2-5) - measures variance versus degrees of freedom for various combinations within the data
-* Use of [propensity scores](https://journalprivacyconfidentiality.org/index.php/jpc/article/view/568)
-* Off-Manifold and Latent Space checks (consider using principal component analysis (PCA) or [t-distributed stochastic neighbor embedding](https://distill.pub/2016/misread-tsne/) (t-SNE) to visualise any new variable combinations created)  
-* Running two statistical/machine learning models related to the end use-case on the raw and synthetic data to compare the resultant summary statistics 
+* **Profile Comparison** and frequency statistics of combinations within the data (e.g. Pearson’s correlations and similarity matrices)
+* **Statistical Distribtuion Comparisons** such as Gaussian mixture log liklihood; Kolmogoroc-Smirnov test for comparing the cumulative distribution function of two continuous variables and the chi-squared test for comparing discrete distributions.
```
Comment (danjscho) [inline synthetic.md:106]: Typo - `liklihood` -> `likelihood`

File: synthetic.md [hunk 2]
```diff
@@ -100,16 +102,18 @@ At this point it is useful to again consider if synthetic data is the right solu
...
+* **Profile Comparison** and frequency statistics of combinations within the data (e.g. Pearson’s correlations and similarity matrices)
+* **Statistical Distribtuion Comparisons** such as Gaussian mixture log liklihood; Kolmogoroc-Smirnov test for comparing the cumulative distribution function of two continuous variables and the chi-squared test for comparing discrete distributions.  
+* **Statistical Outcome Comparisons** such as [Voas Williamson statistic](https://onlinelibrary.wiley.com/doi/abs/10.1002/1099-1220%28200009/10%296%3A5%3C349%3A%3AAID-IJPG196%3E3.0.CO%3B2-5) (measures variance versus degrees of freedom for various combinations within the data); and use of [propensity scores](https://journalprivacyconfidentiality.org/index.php/jpc/article/view/568) (This paper demonstrates the equivalency of these two approaches [Guidelines for Producing Useful Synthetic Data](https://arxiv.org/pdf/1712.04078.pdf)).  Kullback-Leiber divergenece, Gower distance and the Wasserstein metric can all be used to investigate an aggregate difference between the synthetic and ground truth data.
```
Comment (danjscho) [inline synthetic.md:107]: Typo - `divergenece` -> `divergence`

File: synthetic.md [hunk 3]
```diff
@@ -100,16 +102,18 @@ At this point it is useful to again consider if synthetic data is the right solu
...
+* **Off-Manifold and Latent Space checks** - consider using principal component analysis (PCA) or [t-distributed stochastic neighbor embedding](https://distill.pub/2016/misread-tsne/) (t-SNE) to visualise any new variable combinations created
```
Comment (danjscho) [inline synthetic.md:108]: US spelling? - `neighbor` -> `neighbour`

**Review by danjscho — CHANGES_REQUESTED** [review]
Typos to fix, but looks good aside from that.

File: synthetic.md [hunk 4]
```diff
@@ -207,6 +213,14 @@ MIMIC-III is a large, freely-available database comprising deidentified health-r
 
 ### Tools
 
+[SynthVAE](https://github.com/nhsx/SynthVAE)
+
+Fidelity = Medium/High
+
+Technique = Perturbations of the manifold - Variational AutoEncoder with Differential Privacy
+
+An NHSX Data Science Intern Project focussed on developing an open source tool for generating medium fidelity data with high privacy.  USes the PyTorch implmentation of SGD-DP.
```
Comment (danjscho) [inline synthetic.md:222]: `USes` -> `uses`

File: synthetic.md [hunk 5]
```diff
@@ -100,16 +102,18 @@ At this point it is useful to again consider if synthetic data is the right solu
...
+* **End Use Case Comparisons** - Running two statistical/machine learning models related to the end use-case on the raw and synthetic data to compare the resultant summary statistics 
+* **Detection Metrics** - Running a classifer (e.g. Logistic of SVM) on both the synthetic and ground truth data to see how accurately this can distinguish the difference. 
 
 The Synthetic Data Vault project has a section on [Evaluation Framework](https://sdv.dev/SDV/user_guides/evaluation/evaluation_framework.html) built in python which includes a small suite of functions which can be viewed separately or aggregated to give an overall score for the synthetic data.  These functions include some of the statistical tests above as well as likelihood and detection metrics which compare the real and synthetic data when probabilistic and machine learning models are applied.  This implementation is available for single data tables, multi table situations and time series data.
```
Comment (danjscho) [inline synthetic.md:112]: `Synthetic Data Vault` -> `Synthetic Data Vault (SDV)`

File: synthetic.md [hunk 6]
```diff
@@ -133,7 +137,9 @@ Some tests for privacy include:
...
+Although all these tests are useful, none give a definitive metric which can be tested against.  **Differential Privacy** is the most established mathematical way of defining some level of privacy in the data.  This is a large field which won’t be covered here, but instead we refer the reader to an excellent set of resources and active research programme at the van der schaar-lab.  A summary article on their synthetic data thoughts can be found [here](https://www.vanderschaar-lab.com/synthetic-data-breaking-the-data-logjam-in-machine-learning-for-healthcare/).  We are currently reseaching the application of differential privacy using the established Stochastic gradient descent differential privacy and the more recent PATE (Private Aggregation of Teacher Ensembles) implementation.
```
Comment (danjscho) [inline synthetic.md:140]: `Stochastic` -> `stochastic`

**Pull request merged**